docs/Reference/CLI/CLI-Syntax.md

```diff
@@ -1517,6 +1603,133 @@ rpc-http-port="3435"
 Specifies HTTP JSON-RPC listening port (TCP).
 The default is 8545. Ports must be [exposed appropriately](../../HowTo/Find-and-Connect/Configuring-Ports.md). 
 
+### rpc-http-tls-ca-clients-enabled
+
+```bash tab="Syntax"
+--rpc-http-tls-ca-clients-enabled
```

Comment:
The syntax is:  
`--rpc-http-tls-ca-clients-enabled[=<true|false>]`

Reply:
Done.